prow/plugins.yaml

```diff
@@ -157,6 +157,8 @@ slack:
             - maciekpytel # 1.10 patch release manager
         release-1.11:
             - foxish # 1.11 patch release manager
+        release-1.12:
+            - dougm # 1.12 patch release manager
```

Comment:
nit: this should be "1.12 branch manager". patch release is for after GA release